<launch>
	<node name="cinempc_main_node" pkg="cinempc" type="cinempc_main_node"  output="screen">
	</node>
</launch>
